### DISCUSSION TITLE
Will messages from Raspberry Pi conflict with  Flow V2 deck?

### DISCUSSION THREAD

**Discussion by Residualstress** [bitcraze/discussions#1049] [Q&A] [answered]
Dear Sir/Madam,

I want to connect the Raspberry Pi with Crazyflie bolt 2 to send data about the position of the objects in the picture captured by a camera mounted on Raspberry Pi. 
![1](https://github.com/bitcraze/discussions/assets/92587824/118b0a93-bbfd-4f70-8c3e-942b468eac4c)
It recommends using the pins (TX2 & RX2) for UART2. but I also found in the schematic of Flow V2 deck that it also uses TX2 & RX2 to send and receive message.
![2](https://github.com/bitcraze/discussions/assets/92587824/ea813b0a-87e9-4123-a5de-7292c32dcfec)
So my question is that if we connect a Raspberry pi and a Flow V2 deck with using TX2 & RX2 in the same time, will messages from them conflict with each other? 

Thanks!

**Comment by ataffanel** [comment, marked as answer]
These two signal passes through "NM" resistors, NM in that case means "no mount". Those are optional connection and they are not made by default so it will not cause any problems in your case.
